translationstring/tests/test__init__.py

```diff
@@ -77,56 +75,56 @@ def test_interpolate_substitution(self):
         inst = self._makeOne('This is $name version ${version}.',
                               mapping=mapping)
         result = inst.interpolate()
-        self.assertEqual(result, u('This is Zope version 3.'))
+        self.assertEqual(result, u'This is Zope version 3.')
```

Comment:
```suggestion
        self.assertEqual(result, 'This is Zope version 3.')
```

& similar below (pyupgrade could help clean this up)

Reply:
75d6c95